backend/src/main/scala/cromwell/backend/io/WorkflowPaths.scala

```diff
@@ -18,20 +18,21 @@ trait WorkflowPaths extends PathFactory {
   def workflowDescriptor: BackendWorkflowDescriptor
   def config: Config
   
-  protected lazy val executionRootString = config.as[Option[String]]("root").getOrElse("cromwell-executions")
+  protected lazy val executionRootString: String = config.as[Option[String]]("root").getOrElse("cromwell-executions")
```

Comment:
I'm definitely going to need more in real life discussion on this one going to an ADT. I do acknowledge that under the "s/he who last touched it" doctrine, I should know more about this, but in many ways, IntelliJ's auto fixer was the one who last touched this line, not me. ;)